jupyter_book/config.py

```diff
@@ -286,18 +293,19 @@ def yaml_to_sphinx(yaml: dict):
                 sphinx_config["extensions"].append(extension)
 
     local_extensions = yaml.get("sphinx", {}).get("local_extensions")
+    add_paths = []
```

Comment:
can you document what `add_paths` is in the docstring of this function so that other maintainers know what it's doing?

Reply:
thanks @choldgraf this is done.

Reply:
Awesome! One quick question - if we just add to sys.path will it accomplish the same thing? (Eg without having to pass sys.path in and out of this function)?

Reply:
this is what I had before the change -- and it worked for running `jupyter-book` locally but failed the `tests` as the path to the test collection wasn't correct in `sys.path`. It was adding the extension to a path at the root level of the test (rather than including the test set folder). I tracked this down to not resolving relative references when a `srcpath` is provided to jupyter-book. This update ensures the path is relative to `_config.yml` and uses `pathlib` to resolve the relative references that will often be specified in `_config.yml`

Reply:
I needed to pass it up to the calling function as `yaml_path` was available in that context.